tfhe/src/integer/wopbs/test.rs

```diff
@@ -51,88 +51,6 @@ fn make_basis(message_modulus: usize) -> Vec<u64> {
     }
 }
 
-pub fn wopbs_native_crt() {
```

Comment:
Why remove these functions?

Reply:
simply unused, those were experiment functions/test functions and not real functions for users